scripts/ao-install.sh

```
@@ -0,0 +1,166 @@
+
+defaults:
+  runtime: tmux
+  agent: claude-code
+  workspace: worktree
+  notifiers: [desktop]
+
+projects:
+EOF
+
+for pid in $PROJECTS; do
+  echo "  $pid:" >> "$CONFIG_FILE"
```

Shellcheck (0.11.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)